src/tests/ftest/util/scrubber_test_base.py

```diff
@@ -131,3 +132,40 @@ def run_ior_and_check_scruber_status(self, pool, cont, fail_on_warning=True):
         # Just make sure scrubber is working here.
         status = self.verify_scrubber_metrics_value(self.initial_metrics, self.final_metrics)
         return status
+
+    def run_mdtest_and_check_scruber_status(self, pool, cont, mdtest_params):
+        """Run mdtest and get scrubber metrics
+
+        Args:
+            pool (object): Pool object
+            cont (object): Container object within the pool.
+            mdtest_params(list): List comprising of different set of mdtest parameters.
+
+        Returns:
+            status(bool) : True (Scrubber working), False(Scrubber not working)
+        """
+        status = False
```

Comment:
Looks line Line 147-159 are the same as Line 108-120. Maybe we can move them to a helper method?